synapse/models/infotech.py

```diff
@@ -183,6 +246,30 @@ def getModelDefs(self):
                     'doc': 'A vulnerability as designated by a Common Vulnerabilities and Exposures (CVE) number.',
                     'ex': 'cve-2012-0158'
                 }),
+                ('it:sec:cwe', ('str', {'regex': r'^CWE-[0-9]{1,8}$'}), {
+                    'doc': 'NIST NVD Common Weaknesses Enumeration Specification',
+                    'ex': 'CWE-120',
+                }),
+                ('it:mitre:attack:group', ('str', {'regex': r'^G[0-9]{4}$'}), {
```

Comment:
MITRE ATT&CK ID's are not unique so cannot be used as primary properties. #sadbuttrue They may be duplicated (with different meanings) across each ATT&CK "matrix" (currently there are three - Enterprise, Mobile, and ICS). Also, not sure but it's possible that IDs could be changed and / or merged as MITRE makes updates.